src/Plugin/views/display/GraphQL.php

```diff
@@ -94,9 +95,22 @@ protected function defineOptions() {
     unset($options['exposed_block']);
     unset($options['css_class']);
 
+    $options['graphql_query_name'] = ['default' => ''];
     return $options;
   }
 
+  public function getGraphQLQueryName() {
+    $graphql_query_name = strip_tags($this->getOption('graphql_query_name'));
+    if (empty($graphql_query_name)) {
+      $viewId = $this->view->id();
+      $displayId = $this->display['id'];
+      // @todo: fix PHP fatal when graphql_core is not installed.
+      // fix: move this GraphQL display plugin under graphql_views module.
+      $graphql_query_name = graphql_core_camelcase([$viewId, $displayId, 'view']);
```

Comment:
hmm, I am not sure what you mean. What should be the fix here?
Is there another function to camelcase the name? Or the graphql_core_camelcase() function can be moved to graphql module? Or should the function be duplicated inside this file?